src/main/java/ca/utoronto/fitbook/application/port/out/response/ViewExerciseInfoResponse.java

```diff
@@ -0,0 +1,14 @@
+package ca.utoronto.fitbook.application.port.out.response;
+
+import ca.utoronto.fitbook.entity.Exercise;
+import lombok.NonNull;
+import lombok.Value;
+
+import java.util.ArrayList;
+
+@Value
+public class ViewExerciseInfoResponse {
+
+    @NonNull
+    ArrayList<Exercise> exercises;
```

Comment:
I think we should return two lists, one for temporal exercises and one for repetitive exercises.

Reply:
We can achieve this by checking the type of exercises coming out of the repository.